Library/Formula/lha.rb

```diff
@@ -1,14 +1,32 @@
 class Lha < Formula
   desc "Utility for creating and opening lzh archives"
-  homepage "http://lha.osdn.jp"
+  homepage "https://lha.osdn.jp"
   url "http://dl.osdn.jp/lha/22231/lha-1.14i-ac20050924p1.tar.gz"
   version "1.14i-ac20050924p1"
   sha256 "b5261e9f98538816aa9e64791f23cb83f1632ecda61f02e54b6749e9ca5e9ee4"
 
+  head do
+    url "http://scm.osdn.jp/gitroot/lha/lha.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
   def install
+    if build.head?
+      system "aclocal"
+      system "automake", "-a"
+      system "autoheader"
+      system "autoconf"
```

Comment:
Is there not a `./autogen.sh` or `autoreconf` step that handles all of this? Most upstreams put something convenient in place.